Integrations/integration-Kenna.yml

```diff
@@ -0,0 +1,487 @@
+                context.Kenna[i].ScannerVulnerabilities = [];
+                for (var j = 0; j < res[i].scanner_vulnerabilities.length; j++) {
+                    context.Kenna[i].ScannerVulnerabilities[j] = {};
+                    context.Kenna[i].ScannerVulnerabilities[j].ExternalID = res[i].scanner_vulnerabilities[j].external_unique_id;
+                    context.Kenna[i].ScannerVulnerabilities[j].Open = res[i].scanner_vulnerabilities[j].open;
+                    context.Kenna[i].ScannerVulnerabilities[j].Port = res[i].scanner_vulnerabilities[j].port;
+                }
+                context.Kenna[i].Score = res[i].score;
+                if (res[i].service_ticket) {
+                    context.Kenna[i].ServiceTicket = {};
+                    context.Kenna[i].ServiceTicket.DueDate = res[i].service_ticket.due_date;
+                    context.Kenna[i].ServiceTicket.ExternalIdentifier = res[i].service_ticket.external_identifier;
+                    context.Kenna[i].ServiceTicket.Status = res[i].service_ticket.status;
+                    context.Kenna[i].ServiceTicket.TicketType = res[i].service_ticket.ticket_type;
+                }
+                context.Kenna[i].Severity = res[i].severity;
+                context.Kenna[i].Status = res[i].status;
+                context.Kenna[i].Threat = res[i].threat;
+                context.Kenna[i].TopPriority = res[i].top_priority;
+            }
+
+            return {
+                Type: entryTypes.note,
+                Contents: res,
+                ContentsFormat: formats.json,
+                ReadableContentsFormat : formats.markdown,
+                EntryContext : context,
+                HumanReadable: tableToMarkdown('Kenna Vulnerabilities', hr)
+            };
+    }
+
+    function searchVulnerabilitiesRequest(urlQuery) {
+        var requestUrl = url.replace(/[\/]+$/, '') + '/vulnerabilities/search' + encodeToURLQuery(urlQuery);
+        var res = http(
+            requestUrl,
+            {
+                Method: 'GET',
+                Headers: {
+                    'X-Risk-Token': [apiKey],
+                    'Content-Type': ['application/json'],
+                    'Accept': ['application/json']
+                }
+            },
+            params.insecure,
+            params.proxy
+        );
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Request Failed.\nStatus code: ' + res.StatusCode + '.\nBody: ' + JSON.stringify(res) + '.';
+        }
+        return JSON.parse(res.Body);
+    }
+
+
+    function getConnectors() {
+        var response = getConnectorsRequest();
+        var res = response.connectors;
+        var hr = [];
+        var context = {};
+        if (!context.Kenna) {
+            context.Kenna = {};
+        }
+        context.Kenna.ConnectorsList = []
+        for (var i = 0; i < res.length; i++) {
+            hr[i] = {
+                'Host' : res[i].host,
```

Comment:
it will be more readable if you replace `res` => `connectors` 
and in for loop use `var connector = connectors[i]`
in use `connector.running` instead of `res.running`